moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -247,6 +272,7 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
     {
       title: "Project name",
       field: "project_name",
+      hidden: hiddenColumns["project_name"],
```

Comment:
When I run this locally, I get this error on the `/moped/projects` page:
```
ProjectsListViewTable.js:275 Uncaught TypeError: Cannot read properties of null (reading 'project_name')
    at ProjectsListViewTable (ProjectsListViewTable.js:275:1)
```

maybe the object name should be `defaultHiddenColumns` instead of `hiddenColumns`? Or something is weird happening with the way the `hiddenColumns` value get instantiated with `localStorage` in 169? I can show you my error if you need a better demo.

Reply:
Thanks for catching this - I was able to reproduce the bug after manually deleting the object stored in localStorage.